code/Model/EditableFormField/EditableNumericField.php

```diff
@@ -82,4 +82,12 @@ protected function updateFormField($field)
             $field->setAttribute('data-rule-max', $this->MaxValue);
         }
     }
+    public function validate()
+    {
+        $result = parent::validate();
+        if ($this->MinValue > $this->MaxValue) {
+            $result->addError("Minimum length should be less than the Maximum length.");
```

Comment:
As with the other PR, we need to consider location here:

```suggestion
            $result->addError(
                _t(__CLASS__ . '.ORDER_WARNING', 'Minimum length should be less than the maximum length.')
            );
```